.github/workflows/windows-ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: windows-ci
+
+on:
+  [push, pull-request]
+
+jobs:
+  compile:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.5', '3.6', '3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+
+      - run: pip install wheel
+
+      - run: python setup.py build_ext --compiler=msvc --define YY_NO_UNISTD_H
+
+      - run: python setup.py bdist_wheel
```

Comment:
I don't see the point of creating a wheel if the only thing we are checking is that the code compiles. I would drop this and the `pip install wheel` above.

Reply:
add a step `twine check dist/*` to check wheel file.